### DISCUSSION TITLE
Does RNN have a path to adapting to the new architecture of React Native?

### DISCUSSION THREAD

**Discussion by samjayhk** [wix/react-native-navigation#7538] [Q&A] [answered]
Start at version 0.68, React Native has added options for Fabric and Turbo Modules support. As of version 0.69, React Native will implement concurrency features with React v18...

RNN is one of the best RN libraries out there in my opinion, and I also use a lot of RN libraries developed by the WIX team in my company projects two years ago... The only problem at the moment is that RNN doesn't seem to have a roadmap related to the new architecture integration? Since RNN has controlled the entry point of the entire application from the beginning, I must consider whether I need to abandon RNN in the future and move to other libraries to achieve the same function if RNN does not have a relevant roadmap.

So, I am wondering if RNN team have looking into new architecture? Thanks a lot!

**Comment by zcmgyu** [comment]
Track here: https://github.com/wix/react-native-navigation/issues/7466

**Comment by zcmgyu** [comment, marked as answer]
Rolled out

https://github.com/wix/react-native-navigation/pull/7744
